tests/bdd/session/session_test.go

```diff
@@ -0,0 +1,190 @@
+package session_test
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+// ************************************************************************************************************
+// Milestone: Sessions
+//
+// Tentative Timeline (4-6 weeks)
+//
+// Unanswered Questions?
+// - Where is the seed data coming from? 99% sure -> *WORLD STATE*
+// - How is the state of the session nodes maintained?
+// - Is service request validation ddos safe?
+// - Forking behavior
+// ************************************************************************************************************
+
+var _ = Describe("Session", func() {
+
+	Describe("Session Creation \\ Computing", func() {
+
+		Context("Invalid Seed Data", func() {
+			// n-1 Block Hash (probably world state), blockchains requested [list] (probably  world state), node [list] (probably world state))
+
+			Context("Parameters are missing or null", func() {
+
+				It("should error", func() {
+					// Code goes here ...
+				})
+			})
+
+			Context("Devid1 is incorrect...", func() {
+
+				Context("Devid is incorrect format", func() {
```

Comment:
I would remove this layer of context and go directly to the "it"

Reply:
I actually disagree with this one, it would improve brevity, however it reads better as it is right now.